src/lib.rs

```diff
@@ -116,12 +116,14 @@ pub fn wildmon<R: Rng + ?Sized>(
     }
 
     let level = rng.gen_range(1..=100);
+	
+	let mut mon = format!("Wild {}{} (lv{})", name, gender.symbol(), level);
```

Comment:
Speaking of whitespace, can you normalize the indentation? It looks like you're using tabs instead of spaces. :stuck_out_tongue:

Reply:
Changed with new pr